docs/development/releasing.md

```diff
@@ -18,8 +19,10 @@ The prepare release script will do the following:
 1. Check if a release is already in process and if so it will halt.
 2. Create a release branch.
 3. Bump the version number in several files.  See [this bump commit] for an example.
-4. Create a draft release on GitHub with auto-generated release notes
-5. Push the release branch so that the release builds get built.
+4. Create a draft release on GitHub with auto-generated release notes.
+5. Push the release branch.
+6. Call the APIs to run the release builds.
```

Comment:
What does "Call the APIs" mean here?

Reply:
REST calls to trigger CI on the various platforms.